package.json

```
@@ -1,36 +1,24 @@
 {
-  "name": "create-dao",
-  "version": "1.4.3",
-  "private": "true",
-  "description": "",
+  "name": "root",
+  "version": "0.0.0",
+  "private": true,
   "scripts": {
     "dev": "yarn workspace @create-dao/next-app dev",
     "build": "yarn workspace @create-dao/next-app build",
     "start": "yarn workspace @create-dao/next-app start",
     "lint": "yarn workspace @create-dao/next-app lint",
     "chain": "yarn workspace @create-dao/next-app chain",
     "compile": "yarn workspace @create-dao/next-app compile",
-    "deploy": "yarn workspace @create-dao/next-app deploy"
+    "deploy": "yarn workspace @create-dao/next-app deploy",
+    "update-version": "lerna version --force-publish --no-private",
+    "publish-latest": "leran publish from-package",
+    "start:npx:create-dao": "node ./packages/create-dao/index.js"
```

```suggestion
    "create-dao": "node ./packages/create-dao/index.js"
```

imo it makes sense to just call this `create-dao`, as if you're running the actual binary. It doesn't really have anything to do with `start` or `npx`, ya know?